.circleci/config.yml

```diff
@@ -32,56 +43,71 @@ jobs:
             - _build
             - deps
 
-  build_elixir_1_13_otp_24:
-    docker:
-      - image: hexpm/elixir:1.13.1-erlang-24.2-alpine-3.15.0
-    steps:
-      - checkout
-      - <<: *install_hex_rebar
-      - restore_cache:
-          keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
-      - run: mix deps.get
-      - run: mix compile
-
-  build_elixir_1_12_otp_24:
-    docker:
-      - image: hexpm/elixir:1.12.1-erlang-24.0.2-alpine-3.13.3
-    steps:
+     steps:
       - checkout
+      - <<: *install_system_deps
       - <<: *install_hex_rebar
       - restore_cache:
           keys:
-            - v1-mix-cache-{{ checksum "mix.lock" }}
+            - v2-mix-cache-{{ checksum "mix.lock" }}
+      - run: mix hex.organization auth smartrent --key $SMARTRENT_HEX_KEY
       - run: mix deps.get
-      - run: mix compile
+      - run: mix deps.unlock --check-unused
+      - run: mix compile --warnings-as-errors
+      - run: mix format --check-formatted
+      - run: mix credo --all
+      - run: mix test
+      - store_test_results:
+          path: test-junit-report.xml
+      - run: mix docs
+      - run: mix hex.build
+      - run: mix dialyzer
+      - save_cache:
+          key: v2-mix-cache-{{ checksum "mix.lock" }}
+          paths:
+            - _build
+            - deps
```

Comment:
This whole block needs to be removed. Its a duplicate `:steps` key for `build_elixir_1_15_otp_26`